server/server_test.go

```diff
@@ -133,7 +132,8 @@ func TestSecureComm(t *testing.T) {
 	}
 
 	// this makes sure the Prometheus metrics endpoint can start (we made an API call above; there should be metrics)
-	if s, err := getRequestResults(t, httpClient, metricsURL, basicCredentials); err != nil {
+	var s string
```

Comment:
Why not ``if s, err := `` like before? What's the point? It's not shadowed in the else clause.

Reply:
shadowing comes from previous initialization of `err` (see line 101)

Reply:
BTW, I think in that case (as for most error handling cases), this shadowing check is nitpicking. Unfortunately we cannot make a distinction between that kind of nitpicking shadowing, and ones that really brings value, avoids bugs and helps readability.